compiler/src/dotty/tools/dotc/transform/GenericSignatures.scala

```diff
@@ -0,0 +1,510 @@
+package dotty.tools
+package dotc
```

Comment:
Should probably be in `backend/jvm`

Reply:
Debatable where it should go. But since most of it is part of erasure I'd leave it in transform.

Reply:
It is only called from `backend/jvm/DottyBackendInterface.scala`. So my reasoning was that this code is platform specific.
But I don't have a strong opinion about where it should be.